docker/Dockerfile

```diff
@@ -69,6 +71,23 @@ RUN git clone --recursive --branch "${RALIBRETRO_VERSION}" --depth 1 https://git
     make HAVE_CHD=1 -f ./Makefile.RAHasher
 
 
+FROM alpine:${ALPINE_VERSION} AS emulator-stage
+
+RUN apk add --no-cache \
+    7zip \
+    wget
+
+ARG EMULATORJS_VERSION=4.2.0
+wget "https://github.com/EmulatorJS/EmulatorJS/releases/download/v${EMULATORJS_VERSION}/${EMULATORJS_VERSION}.7z" && \
+7z x -y "${EMULATORJS_VERSION}.7z" -o/emulatorjs && \
+rm -rf "${EMULATORJS_VERSION}.7z"; \
+
+ARG RUFFLE_VERSION=nightly-2024-12-28
+ARG RUFFLE_FILE=ruffle-nightly-2024_12_28-web-selfhosted.zip
+wget "https://github.com/ruffle-rs/ruffle/releases/download/${RUFFLE_VERSION}/${RUFFLE_FILE}" && \
+unzip -o "${RUFFLE_FILE}" -d /ruffle && \
+rm -f "${RUFFLE_FILE}"; \
```

Comment:
Are these steps working? We are missing the `RUN` statement in these lines.

Reply:
ah shit nice catch, they were removed during refactor, will fix